lib/job/index.js

```diff
@@ -45,8 +45,10 @@ class Job {
     }
 
     // Set defaults if undefined
+    if (!attrs.nextRunAt && attrs.nextRunAt !== null) {
```

Comment:
this function gets run when you query for jobs.

the original code `attrs.nextRunAt = attrs.nextRunAt : new Date()` will set nextRunAt to `new Date()` if it is `null`

the code change made here does the exact same thing, except it allows nextRunAt to be `null`

Reply:
The problem is that this function also gets run when a new job is created, so it would also allow jobs with `nextRunAt = null`, which may be backwards breaking, no?

Reply:
when a job gets created, nextRunAt will always be `undefined` so it will get set to `new Date()`. otherwise, it will get set to whatever the value was